### ISSUE TITLE
Clicking on an image in the chatbot replicates the image

### ISSUE THREAD

**Issue by salihmarangoz** [gradio-app/gradio#9760]
### Describe the bug

As the title says Clicking on an image in chatbot replicates the image. I checked and there are multiple image objects in the HTML. When I close one of them all of them disappear. Fullscreen view works fine clicking on the image creates this bug.

### Have you searched existing issues?  🔎

- [X] I have searched and found no existing issues

### Reproduction

Go to: https://huggingface.co/spaces/gradio/chatbot_multimodal
Write a few random messages.
Send a message with a picture.
Click on the picture.

### Screenshot

![Peek 2024-10-18 20-15](https://github.com/user-attachments/assets/9b68b5f3-529f-4aae-b00b-6f61f587b145)

### System Info

```shell
Google Chrome browser. The same thing is happening with my project too.
```

### Severity

Blocking usage of gradio

**Comment by freddyaboulton** [comment]
Hi @salihmarangoz - this has been fixed in #9730. The full screen image preview will display properly.

 To be clear, there will still be two different image elements in the HTML but it's not like the image is copied twice.

**Comment by salihmarangoz** [comment]
@freddyaboulton Thanks for the reply!
